test/autocomplete.spec.js

```diff
@@ -195,6 +195,48 @@ describe('Autocomplete', function () {
         expect(dropdown).toBeHidden();
     });
 
+    it('should open the menu if something is written in the textbox', function () {
+        let $input = $('.test-element');
+        let options = {dataSource: test_dataSource};
+        $('.test-element').autocomplete(options);
+
+        let dropdown = $('.autocomplete > ul.items.dropdown-menu');
+
+        expect(dropdown).toBeInDOM();
+        expect(dropdown).toBeEmpty();
+
+        $input.val('a');
+
+        expect($input.val()).toBe('a')
+
+        // fire the input event as if someone was typing
+        fireInputEvent($input[0]);
+
+        expect(dropdown).not.toBeEmpty();

```

Comment:
Does this really test if the dropdown is open?
Shouldn't it test if the dropdown is visible? Afaik `expect(dropdown).not.toBeEmpty();` only tests if there are elements in the DOM and not if they are visible.

Reply:
You are right sire :shipit: